hack/install_swagger.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+BIN="$BINDIR/swagger"
+ARCH=$(go env GOARCH)
+
+die() { echo "${1:-No error message given} (from $(basename $0))"; exit 1; }
+
+function install() {
+    echo "Installing swagger v$VERSION into $BIN"
+    curl --fail-with-body --location -o $BIN \
+        https://github.com/go-swagger/go-swagger/releases/download/v$VERSION/swagger_linux_$ARCH
+    chmod +x $BIN
+    $BIN version
+}
+
+[ -n "$VERSION" ] || die "\$VERSION is empty or undefined"
+[ -n "$BINDIR" ] || die "\$BINDIR is empty or undefined"
+
+if [ ! -x "$BIN" ]; then
+    install
+else
+    if $BIN version; then
+        echo "Using existing $BIN"
```

Comment:
That's not the intention: this will happily use swagger 0.1 even if an explicit 99.99 is requested.

Reply:
Sure.